res/values-pt-rBR/strings.xml

```diff
@@ -68,13 +68,13 @@
     <string name="Difficulty_">"Dificuldade:"</string>
     <string name="Delay_Disconnect">"Atrasar a Desconexão"</string>
 
-    <string name="Max_Players">"Máx. de Jogadores"</string>
-    <string name="Min_Players">"Mín. de Jogadores"</string>
+    <string name="Max_Players">"Máximo de Jogadores"</string>
+    <string name="Min_Players">"Mínimo de Jogadores"</string>
```

Comment:
Lines 71 and 72:

Please keep the abbreviations like it is in English (blue colour)